.github/workflows/file_health.yaml

```diff
@@ -1,11 +1,6 @@
 name: GitHub Actions file health check
 
-on:
-  push:
-    branches: [ '**' ]
-  pull_request:
-    types: [edited, opened, reopened, synchronize]
-    branches: [ '**' ]
```

Comment:
What '**' does mean? Is default the same?

Reply:
>What '**' does mean? 

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

>Is default the same?

I think yes.